### PR TITLE
8257719: JFXPanel scene fails to render correctly on HiDPI after fix for JDK-8199592

### PR DESCRIPTION
**Pull request by jperedadnr**
This PR fixes a regression introduced in [JDK-8199592](https://bugs.openjdk.java.net/browse/JDK-8199592), where scene initialization was moved after updating output scales. However, in case of embedded windows (used by `JFXPanel` or `FXCanvas`), the scene has to be initialized before.

A test has been included. Before applying the fix, the JFXPanel is rendered with a garbled image, and after the fix, the rendering is correct.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8257719](https://bugs.openjdk.java.net/browse/JDK-8257719): JFXPanel scene fails to render correctly on HiDPI after fix for JDK-8199592

### Reviewers
 * [Kevin Rushforth](https://openjdk.java.net/census#kcr) (@kevinrushforth - **Reviewer**)
 * [Johan Vos](https://openjdk.java.net/census#jvos) (@johanvos - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jfx pull/364/head:pull/364`
`$ git checkout pull/364`

### CODE REVIEW

**Review by kevinrushforth — APPROVED** [review]
Looks good with one minor comment.

**Review by kevinrushforth — APPROVED** [review]

File: tests/system/src/test/java/test/robot/javafx/embed/swing/JFXPanelHiDPITest.java [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+import static test.util.Util.TIMEOUT;
+
+public class JFXPanelHiDPITest {
+
+    static CountDownLatch launchLatch;
+
+    private static Timer t;
+    private static MyApp myApp;
+
+    private static final double SCALE = 1.25; // 1.25, 1.5, 1.75
```
Comment (johanvos) [inline tests/system/src/test/java/test/robot/javafx/embed/swing/JFXPanelHiDPITest.java:68]: Can you add some explanation on these values? Why are the other commented values there?

**Review by johanvos — APPROVED** [review]

**Review by kevinrushforth — APPROVED** [review]

**Pull request closed without merging**